src/main/java/com/happeningnow/controller/OrganizerController.java

```diff
@@ -0,0 +1,55 @@
+package com.happeningnow.controller;
+
+import com.happeningnow.model.Organizer;
+import com.happeningnow.service.ServiceOrganizer;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/organizer")
+public class OrganizerController {
+
+    private final ServiceOrganizer serviceOrganizer;
+
+    public OrganizerController(ServiceOrganizer serviceOrganizer){
+        this.serviceOrganizer = serviceOrganizer;
+    }
+
+    @PostMapping("/save")
+    public ResponseEntity<Organizer> save(@RequestBody Organizer organizer){
+        Organizer saveOrganizer = serviceOrganizer.save(organizer);
+        return new ResponseEntity<>(saveOrganizer,HttpStatus.CREATED);
+    }
+
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public ResponseEntity<Organizer> findById(@PathVariable UUID id) {
+        Optional<Organizer> organizer = serviceOrganizer.findById(id);
```

Comment:
This validation should go inside the service.

- Take a look the method .orElseThrow() from Optional, it is really useful when dealing with optional. 
- Instead of throwing a "RuntimeException" I want you to create a dedicated exception "RecordNotFoundException" that extends from exception and takes a String as an argument.
- So, inside Service I expect to see something like 
`public T findById(UUID uuid){
        return repository.findById(uuid).orElseThrow(() -> new RecordNotFoundException("Record not found"));
    }`